packages/ember-glimmer/lib/utils/process-args.js

```diff
@@ -48,15 +50,32 @@ class SimpleArgs {
     for (let i = 0, l = keys.length; i < l; i++) {
       let name = keys[i];
       let value = attrs[name];
+      let ref = namedArgs.get(name);
 
-      args[name] = namedArgs.get(name);
+      if (ref[MUT] && !ref[ACTION]) {
+        attrs[name] = new MutableCell(ref, value);
+      }
+
+      args[name] = ref;
       props[name] = value;
     }
 
     return { attrs, props };
   }
 }
 
+class MutableCell {
+  constructor(ref, value) {
+    this._ref = ref;
+    this[MUTABLE_CELL] = true;
+    this.value = value;
+  }
+
+  update(val) {
+    this._ref[UPDATE](val);
```

Comment:
So I added tests to verify the value of `component.attrs.myAttr.value` and `component.set('myAttr')`and they all seem to be passing without any code changes. Is that what you meant?